roadmap-site/README.md

```diff
@@ -0,0 +1,109 @@
+# Challenges
+
+## User Object
+
+{
+"user_id":<memberid>,
+"first_name":string,
+"last_name":string,
+"yoe": number,
+"company": string,,
+"designation": string,
+"img": string,
+"github_id": string,
+"linkedin_id": string,
+"twitter_id": string,
+"instagram_id": string,
+}
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|  [GET /challenges](#get-challenges)| Returns all challenges          |
+|[POST /challenges](#post-challenges)|       Creates new challenge     |
+|[POST /challenge/subscribe](#post-subscribe) | subscribes users to challenge|
```

Comment:
![image](https://user-images.githubusercontent.com/66248039/97665854-72530680-1aa2-11eb-8a9d-f3051da42414.png)
@sumitd94 Table looks great !! Currently, the "POST /challenge/subscribe" when clicked, is not navigating to the place where actual details of route are mentioned. This is perfectly working for the other 2 routes. Can you just make a little modification to get it working? 

"[POST /challenge/subscribe](#post-subscribe)" if changed to  "[POST /challenge/subscribe](#post-challengesubscribe)"  can fix it. Thank you!

Reply:
Thanks @Rucha1499 , I have fixed this